packages/material/scss/scheduler/_layout.scss

```diff
@@ -145,8 +145,34 @@
 
         .k-edit-form-container { width: 390px; }
 
-        .k-edit-label { width: auto; }
-        .k-edit-field { width: auto; }
+        .k-edit-label {
+            width: auto;
+            margin-bottom: 0.5rem;
+            label {
```

Comment:
Add an empty row between `margin-bottom: 0.5rem;` and `label {` rows. 
Use `em` not `rem`;